README.md

```diff
@@ -2,65 +2,57 @@
 
 Decentralized wallet identification and authorization through message signing and signature validations.
 
+## Usage
+
+Stratis Open Auth Protocol can be used for the following purposes:
+
+* Personalising the data that a dApp stores about and displays to a user
+* Verifying that a user has access to the private key of a wallet address
+
 ## Benefits
 
 - Decentralized authentication
 - No third party access to sensitive user info
 
 ___
 
-## Wallet Implementation
+## Implementation
+
+### Stratis ID URI
+
+A Stratis ID URI can easily be parsed by a wallet. It consists of three required parts:
+
+- The scheme - `sid`
+- The callback - a protocol-relative URL which the wallet will send a HTTPS request
+- The UID - a unique identifier for a request, which is present within the query string of the callback
+
+**Example:** `sid:api.example.com/auth?uid=some-guid`
+
+### Wallet Compatibility
 
 Wallet compatibility relies on the implementation of the following requirements:
 
-- Scan QR code or retrieve Base64 authentication request
-- Decode Base64 string authentication request into:
-    ```JSON
-    {
-        "callbackUrl": "https://api.example.com/auth?connectionId=someConnection",
-        "nonce": "some-guid"
-    }
-    ```
+- Ability to scan QR code or retrieve Stratis ID URI
 - User or wallet should validate the callback url of the authentication request
-- Sign the nonce property value of the decoded request
-- _**POST**_ the signed message and public key to the callbackUrl property value of the decoded string
-    ```JSON
+- Sign the UID query string value of the request
+- _**POST**_ the JSON-encoded signed message and and public key to the callback URI, using HTTPS
+    ```
+    POST https://api.example.com/auth?uid=some-guid
+    Content-Type: application/json
     {
         "signature": "signed-message",
         "publicKey": "public-key"
     }
     ```
```

Comment:
@sondreb @rowandh 

@drmathias is proposing removing the nonce altogether and having an arbitrary string be signed that has no meaning to the wallet. Please discuss this area between yourselves on if the wallets should have a meaning message to sign or if it doesn't matter. 

Also, my 2 cents, I lean towards going back to the nonce with a separate and unique `uid` to identify the users connection we'll be sending a token back too.

Reply:
Update: I change my mind.

I like the idea of using query strings and having the entire callback URL be the signed message.